services/common/src/main/java/com/linkedin/openhouse/common/Constants.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.openhouse.common;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Constants {
```

Comment:
shared constants for use in logic and tests

Reply:
good point, i moved the class to only be used in the tables MvcConfig context, but couldn't make fields package private due to unittests in a diff package using these fields